src/main/java/uk/gov/hmcts/reform/bulkscan/utils/BulkScanValidationUtil.java

```diff
@@ -13,14 +14,19 @@ private BulkScanValidationUtil() {
 
     }
 
-    public static boolean isDateValid(String dateStr, String format) {
+    public static boolean isDateValid(String fieldName, String dateStr, String format) {
         DateFormat sdf = new SimpleDateFormat(format, Locale.UK);
         sdf.setLenient(false);
         try {
-            sdf.parse(dateStr);
+            Date parse = sdf.parse(dateStr);
+            Date current = new Date();
+            if ((fieldName.contains("dateOfBirth") || fieldName.contains("dob")) && parse.after(current)) {
```

Comment:
use contants for "dateOfBirth" and "dob"

Reply:
moved to constants